### ISSUE TITLE
crashed on sync.cc line 88

### ISSUE THREAD

**Issue by 0oneo** [grpc/grpc#14270]
### Should this be an issue in the gRPC issue tracker?
 
YES
 
### What version of gRPC and what language are you using?
GRPC: 1.8.4
language: Objective-C
 
 
### What operating system (Linux, Windows, …) and version?
iOS
 
### What runtime / compiler are you using (e.g. python version or version of gcc)
Clang
  
### What did you do?
test GRPC online to profile the statbility
 
### What did you expect to see?
not crash
 
### What did you see instead?
crash on low rate
 
Crashed: com.apple.root.default-qos
EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x0000000000000000

```
Crashed: com.apple.root.default-qos
0  myapp                          0x1037f08f4 gpr_ref_non_zero (sync.cc:88)
1  myapp                          0x1037ee0b8 grpc_connected_subchannel_ref (subchannel.cc:170)
2  myapp                          0x1037d9a1c pf_pick_locked(grpc_exec_ctx*, grpc_lb_policy*, grpc_lb_policy_pick_args const*, grpc_connected_subchannel**, grpc_call_context_element*, void**, grpc_closure*) (pick_first.cc:182)
3  myapp                          0x1037aca1c pick_callback_start_locked(grpc_exec_ctx*, grpc_call_element*) (client_channel.cc:1145)
4  myapp                          0x1037ac66c start_pick_locked(grpc_exec_ctx*, void*, grpc_error*) (client_channel.cc:1306)
5  myapp                          0x1037af588 grpc_combiner_continue_exec_ctx (combiner.cc:261)
6  myapp                          0x1037bae94 grpc_exec_ctx_flush (exec_ctx.cc:93)
7  myapp                          0x1037b877c pollset_work(grpc_exec_ctx*, grpc_pollset*, grpc_pollset_worker**, long) (ev_poll_posix.cc:1057)
8  myapp                          0x1037b0840 cq_next(grpc_completion_queue*, gpr_timespec, void*) (completion_queue.cc:952)
9  myapp                          0x103794450 __27-[GRPCCompletionQueue init]_block_invoke_2 (GRPCCompletionQueue.m:56)
10 libdispatch.dylib              0x182a9aa54 _dispatch_call_block_and_release + 24
11 libdispatch.dylib              0x182a9aa14 _dispatch_client_callout + 16
12 libdispatch.dylib              0x182aa1bc8 _dispatch_queue_override_invoke$VARIANT$mp + 716
13 libdispatch.dylib              0x182aa7cf4 _dispatch_root_queue_drain + 600
14 libdispatch.dylib              0x182aa7a38 _dispatch_worker_thread3 + 120
15 libsystem_pthread.dylib        0x182d4306c _pthread_wqthread + 1268
16 libsystem_pthread.dylib        0x182d42b6c start_wqthread + 4
```
 
### Anything else we should know about your project / environment?
no

**Comment by srini100** [comment]
@0oneo please post your steps to reproduce this crash.

**Comment by 0oneo** [comment]
@srini100 the crash happens on low rate on our production. On my dev machine, I have never came to this crash

**Comment by 0oneo** [comment]
@srini100 the crash happens on low rate on our production. On my dev machine, I have never came to this crash

**Comment by srini100** [comment]
@0oneo Fix for #13327 is in 1.9.x. If you can, please try this version. After some mileage, fix may be backported to 1.8.x.
